index.html

```diff
@@ -217,5 +217,32 @@ <h4>Skills</h4>
     <hr class="skills">
 </section>
 </main>
+
+<!-- Footer Section -->
+
+<footer class="contact-forms">
+    <div class="contact">
+        <h2 class="form-heading">Contact me</h2> 
+    </div>
+    <div class="form-paragraph">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="https://formspree.io/f/mpzbjyrv" method="post">
+        <div class="name">
+            <input name="usr_name" class="name-text" type="text" placeholder="Yeremias" maxlength="30" required >
+        </div>
+        <div class="email">
+            <input name="usr_email" class="email-field" type="email" placeholder="natajayanj14@gmail.com" required>
+        </div>
+        <div class="text-field">
+            <textarea class="text-fld" name="text-field" id="text-field" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+        </div>
+        <div class="form-btn-1">
+            <button type="submit" class="form-btn">Get It Touch</button>
```

Comment:
- [ ] I noticed that you added hover and active (pressed) button interactions for the form submission button. But the colors do not match the defined colors in Figma. Please set the hover color to `#6070FF` and prssed color to `#2230D2`.